config/wdio/wdio.conf.js

```diff
@@ -24,6 +24,10 @@ const ci = process.env.CI;
 /* Use to bail fast while running locally. */
 const bail = process.env.WDIO_BAIL !== 'false' && (process.env.WDIO_BAIL || ci);
 
+/* Use to disabled test assertions on the screenshot(s) comparison results. */
```

Comment:
Should this be `disable` instead of `disabled`? This is pretty minor. It's spelled this way in each of the comment/doc updates.

Reply:
looks right to me...